cmake/modules/SetUpMacOS.cmake

```diff
@@ -85,6 +85,17 @@ if (CMAKE_SYSTEM_NAME MATCHES Darwin)
 
      set(CMAKE_C_LINK_FLAGS "${CMAKE_C_LINK_FLAGS} -bind_at_load -m64")
      set(CMAKE_CXX_LINK_FLAGS "${CMAKE_CXX_LINK_FLAGS} -bind_at_load -m64")
+     
+     if(asan)
+       # See also core/sanitizer/Readme.md for what's happening.
+
+       #This should be the right way to do it, but clang 10 seems to have a bug
+       #execute_process(COMMAND ${CMAKE_CXX_COMPILER} --print-file-name=libclang_rt.asan_osx_dynamic.dylib OUTPUT_VARIABLE ASAN_RUNTIME_LIBRARY OUTPUT_STRIP_TRAILING_WHITESPACE)
```

Comment:
hmmm :( did you open bug report? @Axel-Naumann what could be done here?

Reply:
No bug report. Had enough other things to do. Just wanted to get this working.